### ISSUE TITLE
Update failed then Code wouldn't start.

### ISSUE THREAD

**Issue by bsdaddict** [microsoft/vscode#98186]
Update to 1.45.1 failed and Code wouldn't start afterwards. I set update.enableWindowsBackgroundUpdates to false, but as Code wouldn't start I added that to settings.json. Didn't help code to start. Downloaded 1.45.1 installer, that couldn't remove code.exe. Uninstalled VSCode, 1.45.1 installer still had problems removing code.exe. Treid to delete code.exe manually, access denied. Attempted to take ownership of code.exe, access denied. 

The update error is: 
	An error occurred while trying to rename a file in the destination directory:
	MoveFile failed; code 183
	Cannot create a file when that file already exists.

Updates have gone swimmingly in the past. Whatever changed to bork this one needs to be rolled back, just not sure if it's a VSCode problem or an OS problem. I'm done fighting with this for today, it's already taken up too much of my time. Will use a different editor until the next update when hopefully this problem is fixed.

- VSCode Version: 1.45
- OS Version: 10 Pro x64 18363.836

Steps to Reproduce:

1. Attempt 1.45.1 update
2. (2 hours later) give up, get back to work using a different editor.

Does this issue occur when all extensions are disabled?: Can't tell since MS has seemed to forget who owns this computer.

	D:\Apps\Microsoft VS Code>code --disable-extensions
	Access is denied.

**Comment by bsdaddict** [comment]
Update: had another crack at it today. Just before running the 1.45.1 installer again something about its filename made me wonder... Went back to the download page to check on my hunch and, sure enough, I had downloaded the wrong installer. Seems the big, blue "Download for Windows" button on the VS Code home page grabs the User Installer by default, and that buttons dropdown only lists the one Windows installer. Anyone not installing under appdata has to go to the actual download page to see that there's a optional System Installer. After downloading that one the installer had no complaints regarding being unable to remove code.exe and the reinstall went smoothly.   

Still no clue as to why the initial update attempt failed so spectacularily when prior updates were painless, but at least the reinstall will fix it.  So long as one is using the correct installer, that is. On that note.., I typically check to make sure I'm getting the 64bit version of whatever it is I happen to be downloading.  If that Download for Windows button dropdown had included the system installer it would have been obvious which one I needed to download. I'd like to suggest adding it. Not for me as I'm sure I won't make that mistake again, but I'm also sure that I won't be the last one to make it. Who knows how much grief will be avoided with such a trivial change to the site.
